pkg/dataplane/controller_test.go

```diff
@@ -274,6 +282,11 @@ table inet kube-network-policies {
 		ip6 saddr @podips-v6 queue flags bypass to 102
 		ip6 daddr @podips-v6 queue flags bypass to 102
 	}
+	chain forward-established {
+		type filter hook forward priority filter - 5; policy accept;
+		ct state established ip saddr . th sport . ip daddr . th dport . meta l4proto @block-tuple-v4 reject with icmp port-unreachable
+		ct state established ip6 saddr . th sport . ip6 daddr . th dport . meta l4proto @block-tuple-v6 reject with icmpv6 addr-unreachable
+	}
```

Comment:
we need to do something like this , is the only way I find out we can "break" existing connections https://lore.kernel.org/netfilter-devel/20250313231341.3040002-1-aojea@google.com/T/#m44c410cd56bdc62cc717798a0e28ee4dcaaab9ad